README.md

```diff
@@ -84,6 +84,74 @@ The playbook executes the following steps for you:
+| `oc_password`                 | OpenShift user password.                                               |                    |                  
+| `jenkins_user`                | Jenkins admin username                                                 | admin              |
+| `jenkins_pass`                | Jenkins admin password                                                 | password           |
+| `master_memory_limit`         | Maximum amount of memory the Jenkins master container can use          | 3Gi                |
+| `master_volume_capacity`      | Volume space available for data                                        | 40Gi               |
+
+
+##### Proxy related
+
+If connections need to go over a proxy in the containers, fill these variables.
+
+|  Variable                     | Description                                                           |
+| ----------------------------- | --------------------------------------------------------------------- |
+| `proxy_host`                  | Proxy hostname                                                        |
+| `proxy_port`                  | Proxy port                                                            |
+| `proxy_user`                  | Proxy username                                                        |
+| `proxy_pass`                  | Proxy password                                                        |
+| `proxy_protocol`              | Proxy protocol                                                        |
+                                                                                                        
+##### IOS/OSX related
+
+These are OSX related variables. Feel free to keep them blank if you are not planning
+to build IOS applications.
+
+|  Variable                     | Description                                                                      |          |
+| ----------------------------- | -------------------------------------------------------------------------------- | -------- |
+| `macos_user`                  | Which OS user to use to connect to OSX machines                                  | jenkins  |
+| `ansible_become_pass`         | Password required to do sudo for the SSH user on OSX machines                    |          |
+| `xcode_install_user`          | OSX user to install Xcode. If empty, Xcode won't be installed                    |          |
+| `xcode_install_password`      | Password of OSX user to install Xcode                                            |          |
+| `credential_private_key_path` | Path to the private key that Jenkins should use to SSH into the macOS node       |          |
+| `credential_passphrase`       | Passphrase for the private key above                                             |          |
+| `credential_public_key_path`  | Public key of the private key above. To be added to Jenkins' authorized keys     |          |
```

Comment:
I think this may have changed somewhat actually. See https://github.com/aerogear/aerogear.org/blob/Build-Farm-2/docs/digger/installation/macos-options.asciidoc#which-command-should-be-used-to-change-the-following-variables